### PR TITLE
OKTA-588627: loginpage odyssey style

### PR DESCRIPTION
**Pull request by shuowu-okta**
## Description:

Style login page to follow odyssey style

## PR Checklist

- [ ] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-588627](https://oktainc.atlassian.net/browse/OKTA-588627)

### Screenshot/Video:

Design review: https://oktainc.atlassian.net/browse/OKTA-588627?focusedCommentId=4528866

### CODE REVIEW

File: src/v3/src/components/AuthContainer/AuthContainer.tsx
Comment (shuowu-okta) [inline src/v3/src/components/AuthContainer/AuthContainer.tsx:57]: moved to style.scss to let loginpage control widget position

File: src/v3/src/components/Widget/index.tsx
Comment (shuowu-okta) [inline src/v3/src/components/Widget/index.tsx:514]: remove redundant ScopedCssBaseline

File: src/v3/src/components/Widget/style.scss
Comment (shuowu-okta) [inline src/v3/src/components/Widget/style.scss:226]: `!important` is used to override styles from loginpage.theme.css

File: yarn.lock
Comment (trevoring-okta) [inline yarn.lock:13989]: We've seen this `json5` dependency randomly change on some `yarn installs` and not on others and we can't figure out why. Could you try reverting this and seeing if you can get an updated `yarn.lock` that does not have this dependency change?

File: src/v3/src/components/Widget/style.scss
Comment (trevoring-okta) [inline src/v3/src/components/Widget/style.scss:1]: Could you attach some screenshots so we can see what the styling looks like?

Comment (trevoring-okta) [reply src/v3/src/components/Widget/style.scss:1]: Oh I can see some screenshots in the JIRA you linked but might be nice to have here so we can discuss as part of PR review. I'm anticipating the `vrt-v3` Bacon suite might fail and you'll have to update the base screenshots for VRT since it seems like some fonts have changed within the widget container?

**Review by lesterchoi-okta — COMMENTED** [review]
See font/i18n questions. Do you think it'd be smart to separate _actual_ widget styles from css we're writing to compensate for loginpage, EUD, EUS, etc. so we can move them later?

File: src/v3/src/components/Widget/style.scss
Comment (shuowu-okta) [reply src/v3/src/components/Widget/style.scss:229]: Update it in `loginpage.theme.css` can affect styles for loginpage that still stay with gen2, as the theme styles comes after widget styles, right now I didn't find a good approach to prevent gen2 related theme styles.

File: src/v3/src/components/Widget/style.scss
Comment (shuowu-okta) [reply src/v3/src/components/Widget/style.scss:52]: It's following [styles from odyssey](https://github.com/okta/odyssey/blob/4a4a98d1b9e421122332cb0635d823189ee4e72b/packages/odyssey-react-mui/src/theme/components.tsx#L990-L1012). We probably can duplicate the implementation in loginpage as the first step, then once we have decoupled loginpage from monolith, we can directly import the baseline styles into the page.

File: src/v3/src/components/Widget/index.tsx
Comment (shuowu-okta) [reply src/v3/src/components/Widget/index.tsx:514]: Already there

File: src/v3/src/components/Widget/style.scss
Comment (shuowu-okta) [reply src/v3/src/components/Widget/style.scss:1]: Thanks for the reminder, I can upgrade those baselines once get approval on this PR (no further visual changes)

**Comment by shuowu-okta** [comment]
> See font/i18n questions. Do you think it'd be smart to separate _actual_ widget styles from css we're writing to compensate for loginpage, EUD, EUS, etc. so we can move them later?

Currently I think it's mostly separated with the only exception [here](https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/components/Widget/style.css#L169), just copied it over during css to scss migration, and seems like there is already a ticket in backlog to handle it.

File: src/v3/src/components/Widget/style.scss
Comment (lesterchoi-okta) [reply src/v3/src/components/Widget/style.scss:52]: I think [the link](https://github.com/okta/odyssey/blob/4a4a98d1b9e421122332cb0635d823189ee4e72b/packages/odyssey-react-mui/src/theme/components.tsx#L990-L1012) answers the question. Looks like Aeonik is only for headings, Inter for body, and Noto as fallback for non-latin. Which should mean all scenarios are covered by L28 except those that need to be specified as `Noto Sans ___` variants as you have here.

**Review by lesterchoi-okta — APPROVED** [review]

**Review by lesterchoi-okta — APPROVED** [review]
VRT diffs and snapshots reviewed

**Pull request merged**
